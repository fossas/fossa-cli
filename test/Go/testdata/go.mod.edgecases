// dummy comments
//
// can vary in length which can break parsers.
module test/package

go 1.12

toolchain go1.21.1

require repo/name/A v1.0.0 // indirect

require (
	repo/B v0.0.5-0.20190714195934-000000000002
	repo/C v1.1.0
	repo/name/D v4.0.0 // indirect
	repo/E v8.0.0+incompatible
	repo/F_underscore v1.0.0
)

replace repo/B => alias/repo/B v0.1.0

replace (
	repo/C => alias/repo/C v0.0.0-20180207000608-000000000003
	repo/E => alias/repo/E v0.0.0-20170808103936-000000000005+incompatible
    foo v0 => ../foo
    bar => /foo/bar/baz
    repo/F_underscore v1.0.0 => repo/F_underscore v2.0.0
)

exclude repo/B v0.9.0

exclude (
    repo/C v1.0.0
    repo/name/D v3.0.0
)
