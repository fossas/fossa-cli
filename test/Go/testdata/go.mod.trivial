module "github.com/my/package"

require (
    github.com/pkg/one v1.0.0
    github.com/pkg/two/v2 v2.0.0
    github.com/pkg/three/v3 v3.0.0
)

replace github.com/pkg/two/v2 => github.com/pkg/overridden overridden
