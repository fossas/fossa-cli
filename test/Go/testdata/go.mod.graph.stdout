main dep@v0.5.0
dep@v0.5.0 depB@v0.6.0